docs/src/_includes/components/language-switcher.html

```diff
@@ -14,8 +14,11 @@
         <select name="language selector" id="language-select" aria-describedby="language-infobox"
             class="c-custom-select switcher__select">
             {% for key, other_site in sites %}
-            <option value="{{ other_site.language.code }}" data-url="https://{{ other_site.hostname }}" {% if
-                site.language.code==other_site.language.code %} selected {% endif %}>
+            <option
+                value="{{ other_site.language.code }}"
+                data-url="https://{{ other_site.hostname }}/docs{% if process.env.BRANCH !== '/latest' %}/latest{% endif %}{{ page.url | prettyURL }}"
```

Comment:
```suggestion
                data-url="https://{{ other_site.hostname }}/docs/latest{{ page.url | prettyURL }}"
```

This conditional seems unnecessary. If we always want a direct link to the latest version in another language, then it should be the same when the currently browsed version is the latest version?